### ISSUE TITLE
directories in step-by-step guide vignette are inconsistent

### ISSUE THREAD

**Issue by sieste** [stan-dev/rstantools#26]
The instructions in the vignette jump back and forth between assuming that `rstanlm` is the working directory and assuming that `rstanlm` is **in** the working directory.

```r
list.files(all.files = TRUE)
DES <- readLines("DESCRIPTION")
roxygen2::roxygenise(clean=TRUE)
devtools::install(local=FALSE)
```

versus

```r
cat(readLines("rstanlm/Read-and-delete-me"), sep = "\n")
file.remove('rstanlm/Read-and-delete-me')
# // Save this file as rstanlm/src/stan_files/lm.stan
# Save this file as `rstanlm/R/lm_stan.R`
pkg_file <- readLines(file.path("rstanlm", "R", "rstanlm-package.R"))
```

**Comment by jgabry** [comment]
Thanks, looks like I overlooked a few of those. Should be fixed now.

**Comment by jgabry** [comment]
Oops. Thanks! Fixed in 289a79e
